packages/xrpl/src/models/methods/accountObjects.ts

```diff
@@ -59,20 +37,6 @@ export interface AccountObjectsRequest
   marker?: unknown
 }
 
-/**
- * Account Objects can be a Check, a DepositPreauth, an Escrow, an Offer, a
- * PayChannel, a SignerList, a Ticket, or a RippleState.
- */
-export type AccountObject =
```

Comment:
you could keep this the same and have it be as subset of LedgerEntry

Reply:
Looks like that might be necessary:
```
test/integration/transactions/escrowCancel.test.ts:72:42 - error TS2339: Property 'PreviousTxnID' does not exist on type 'LedgerEntry'.
xrpl:       Property 'PreviousTxnID' does not exist on type 'Amendments'.
xrpl:     72           transaction: accountObjects[0].PreviousTxnID,
xrpl:                                                 ~~~~~~~~~~~~~
```

Reply:
Also see how we solved this with the AMM Object.  Part of this issue stems for the fact that most ledger objects have PreviousTxnID.  Actually all of them that an account owns does except for the newly added AMM one.  So we had to solve this same issue with its type by using never.